LICENSES.txt

```diff
@@ -201,7 +201,7 @@ Apache-2.0
   parboiled-core-1.2.0.jar
   parboiled-scala_2.12-1.2.0.jar
   picocli-4.6.1.jar
-  reactor-core-3.4.15.jar
+  reactor-core-3.6.5.jar
```

Comment:
What are these bumps?

Reply:
Auto generated. I assume we have wildcard version on some dependencies so they can change on every build :(.